### PR TITLE
Functional tests login jobs

### PR DESCRIPTION
**Pull request by ilyassjaatit**
refs #35 

Test API Endpoints:
1. Login
2. Get jobs

### CODE REVIEW

**Review by francescarpi — APPROVED** [review]

**Comment by francescarpi** [comment]
Hi! Could you execute the pre-commit command and push it again, pls?

Thanks!

File: tests/accounts/api/test_auth_login.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+@pytest.fixture
+def create_user() -> UserFactory:
+    user = UserFactory(email=EMAIL, is_active=True)
+    user.set_password(PASSWORD)
+    user.save()
+    return user
+
+
+@pytest.fixture
```
Comment (jbagot) [inline tests/accounts/api/test_auth_login.py:23]: I would like to add this fixture to `conftest.py` file in the root of tests folder, then all the tests will be able to use it, not only the tests in this file.

**Review by jbagot — COMMENTED** [review]
Really nice job. Thank you to add some tests :)

**Comment by jbagot** [comment]
Black is not passing. Can you execute `black` and push it another time?

**Comment by francescarpi** [comment]
Sorry, there are one conflict. After resolve it, I'll accept the PR.

**Comment by francescarpi** [comment]
Check code in master. The # 149 already introduces this features. In this case, you could close this PR.

Sorry for inconveniences.

**Pull request closed without merging**